tasks/lint.js

```diff
@@ -1,12 +1,12 @@
+import browserSync from 'browser-sync';
+import eslint from 'gulp-eslint';
 import gulp from 'gulp';
 import gulpIf from 'gulp-if';
-import eslint from 'gulp-eslint';
-import browserSync from 'browser-sync';
 
 function lint() {
   return gulp
     .src(['src/js/**/*.js', '!node_modules/**'])
```

Comment:
I wasn't able to find a way that gets this working correctly, so I left it as it was.

Reply:
What was going on here that you wanted to change?

Reply:
Whenever I ran `gulp lint` it never addressed any of the issues in the files. It also seems to require a file path style than the formatting task. Within the formatting task, I'd use `./src/js/**/*.js` and in here it's `src/js/**/*.js`.